src/helpers.test.js

```diff
@@ -6,95 +6,172 @@ import {
     getDefaultValueForType,
     getDefaultValueFromOptions,
     getFunctionName,
+    removeFirstKeyIfNumber,
+    getErrorIndexFromKeys,
 } from './helpers';
 
 describe('helpers', () => {
-    it('should create object from another object with specific keys', () => {
-        const bar = {
-            foo: 'bar',
-            foo2: 'bar2',
-            foo3: 'bar3',
-            foo4: 'bar4',
-        };
-        const foo = pick(bar, ['foo', 'foo2']);
+    describe('pick', () => {
+        it('should create object from another object with specific keys', () => {
+            const bar = {
+                foo: 'bar',
+                foo2: 'bar2',
+                foo3: 'bar3',
+                foo4: 'bar4',
+            };
+            const foo = pick(bar, ['foo', 'foo2']);
 
-        expect(foo).toEqual({
-            foo: 'bar',
-            foo2: 'bar2',
+            expect(foo).toEqual({
+                foo: 'bar',
+                foo2: 'bar2',
+            });
         });
     });
+    describe('difference', () => {
+        it('should compere two array and return difference', () => {
+            const foo = ['foo', 'foo2', 'foo3', 'foo4'];
+            const bar = ['foo', 'foo2'];
 
-    it('should compere two array and return difference', () => {
-        const foo = ['foo', 'foo2', 'foo3', 'foo4'];
-        const bar = ['foo', 'foo2'];
-
-        expect(difference(foo, bar)).toEqual(['foo3', 'foo4']);
+            expect(difference(foo, bar)).toEqual(['foo3', 'foo4']);
+        });
     });
-
-    it('should wrap value to array', () => {
-        const foo = 'test';
-
-        expect(wrapToArray(foo, true)).toEqual(['test']);
-        expect(wrapToArray(foo)).toEqual('test');
+    describe('wrapToArray', () => {
+        it('should wrap value to array', () => {
+            const foo = 'test';
```

Comment:
I try to avoid the word "test" in tests, because I do not know is this keyword important or not. Please, change to `const foo = 'fooValue';` or the whole test to `expect(wrapToArray('foo', true)).toEqual(['foo']);`

Reply:
changed to `fooValue`